mitodl/open-discussions
# Add REST API for search

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1332 

#### What's this PR do?
Adds a REST endpoint which will handle Elasticsearch queries

#### How should this be manually tested?
Run `./manage.py recreate_index` first to make sure your index is up to date.

Go to `/api/v0/search/`. You should see a message saying `GET` isn't supported and a form to make a `POST` request. Type `{}` in the textarea and click POST. You should see a JSON representation of some hits corresponding to the data your user can see.
